src/unit-test/instructions.test.js

```diff
@@ -0,0 +1,27 @@
+import '../setupTest.js'
+import Instructions from '../components/Instructions';
+import fetch from '../unit-test/mocks/instructionFetch';
+global.fetch = fetch;
+
+it('renders without crashing', () => {
+  shallow(<Instructions />)
+});
+
+describe('#componentDidMount', () => {
+  test('calls fetch once called', () => {
+    const fetchSpy = jest.spyOn(window, 'fetch');
+    const setState = jest.fn()
+    const wrapper = shallow(<Instructions id={1}/>);
+    wrapper.instance().componentDidMount()
+    expect(fetchSpy).toBeCalled();
+  })
+});
+
+// describe('#componentWillUnmount', () => {
```

Comment:
Why is this commented out?

Reply:
Not sure how/ if to test this function. For now we will push ahead and if we have time we'll come back around to test it.

Reply:
If it has 100% we might not need it.
I see your point let's move ahead.